scripts/gatk_plot_annotations.R

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env Rscript
+
+# plotting.R script loads ggplot and gridExtra libraries and defines functions to plot variant annotations
+
+library(ggplot2)
+library(gridExtra)
+library(readr)
+
+# Function for making density plots of a single annotation
+makeDensityPlot <- function(dataframe, xvar, split, xmin=min(dataframe[xvar], na.rm=TRUE), xmax=max(dataframe[xvar], na.rm=TRUE), alpha=0.5, log10=FALSE) {
+    if(missing(split)) {
+        plot = ggplot(data=dataframe, aes_string(x=xvar)) + xlim(xmin,xmax) + geom_density()
+        if (log10) {
+            plot = plot + scale_x_log10()
+        }
+        return(plot)
+    }
+    else {
+        return(ggplot(data=dataframe, aes_string(x=xvar, fill=split)) + xlim(xmin,xmax) + geom_density(alpha=alpha) )
+    }
+}
+
+args <- commandArgs(trailingOnly=TRUE)
+
+if (length(args) < 3) {
```

Comment:
Add that having more args populates `input_fields` with functionality of picking annotations?

Reply:
Added a usage doc to the top of the script.